public/data/shukla007.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Abhishek Shukla",
+    "bio": "Full-Stack developer and Open source contributor",
+    "avatar": "https://github.com/shukla007.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "http://github.com/shukla007",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "http://twitter.com/avi0_07",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/abhishek-shukla-682908211/",
+        "icon": "linkedin"
+      }
+    ]
+    
```

Comment:
```suggestion
```

Reply:
what does profile addition mean and what changes are required.

Reply:
LinkFree has profiles of lots of user. Profile addition tag allows the maintainers to know that this is a PR for addition of profile. The changes requested here are to remove the empty line (line no 22)

Reply:
okk Thanks a lot